apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/cells/value-cell/inputs/json-input/json-editor.tsx

```diff
@@ -0,0 +1,294 @@
+// disabled simple-import-sort, because `Prism` needs to imported first here, otherwise it throws an error
+// eslint-disable-next-line simple-import-sort/imports
+import Prism from "prismjs";
+import "prismjs/components/prism-json";
+import "prismjs/components/prism-json5";
+
+import { Box, GlobalStyles } from "@mui/material";
+import {
+  ChangeEvent,
+  KeyboardEvent,
+  useCallback,
+  useEffect,
+  useRef,
+} from "react";
+
+interface JsonEditorProps {
+  value: string;
+  onChange: (value: string) => void;
+  height: string | number;
+}
+
+export const JsonEditor = ({ onChange, value, height }: JsonEditorProps) => {
```

Comment:
This is almost an exact copy of  the `json-editor` at block-protocol repository. https://github.com/blockprotocol/blockprotocol/blob/3e6e44a95c0d9b9bd9f0f8f5efc165e5fafb151e/apps/site/src/components/pages/hub/json-editor.tsx